codazen/webDiplomacy
# Feature/fbfw 143 transition to next phase

To test:

Upon refreshing or page load, loading screen should render. Likewise, when everyone is ready, run gamemaster and the react app should transition to the next phase while displaying the loading screen.

Due to time constraints, fake loading times was not implemented. WDUI is also not rendered during transitions to prevent actions being taken.

https://drive.google.com/file/d/1-27Twg0PzmPhREx-0RxSbrA9BMpoUlm0/view?usp=sharing
